webviewer-wpf/tools/copy-webviewer-files.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs-extra');
+
+const copyFiles = async () => {
+  try {
+    await fs.copy('./node_modules/@pdftron/webviewer/public', './webview2/webviewer/lib');
```

Comment:
Can we change this to `/lib/webviewer`? Just to be consistent with all our other samples